components/admin/rush/CreateDrawer.tsx

```diff
@@ -75,11 +75,13 @@ const CreateDrawer: React.FC<CreateDrawerProps> = ({ onClose }) => {
         <div className="mb-4">
           <div className="mb-2 block">
             <Label htmlFor="graduationYear" value="Name" />
+            <span className="text-red-500"> *</span>
           </div>
           <TextInput
             key="timeframeName"
             required
             id="timeframeName"
+            placeholder="Fall 2024"
```

Comment:
I wonder if there's a way to make a dynamic place holder depending on datetime (like jan to june = Spring + Year, july to dec = Fall + Year)

Reply:
Just a thought haha so we don't need to modify this everytime